### PR TITLE
Added project for the country austria

### PR DESCRIPTION
**Pull request by jimmylevell**
:wave: I did some awesome work for the Pelias project and would love for everyone to have a look at it and provide feedback.  
For a project within Austria we are planning to set up our own pelisa server. The service will be running in a docker image.  
Therefore this existing project needed to be extended to only download the regional data important for Austria. 

Just let me know if I have to add anything more to fulfill your requirements. 

---
#### Here's the reason for this change :rocket:
I have extended the project for the country Austria.  
This enables people to build pelias only for Austria which lowers the computational requirements for hosting their own instance of pelisa.

---
#### Here's what actually got changed :clap:
- changed the pelias.jsonto download the opendata required for Austria
- provided example test link

---
#### Here's how others can test the changes :eyes:
- pull the project and execute the Austrian definition
- execute example URL

### CODE REVIEW

File: projects/austria/.env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+COMPOSE_PROJECT_NAME=pelias
+DATA_DIR=./data
+DOCKER_USER=0
```
Comment (missinglink) [inline projects/austria/.env:3]: We will shortly merge https://github.com/pelias/docker/pull/220 which will prevent users running as root.

This value should either be set as the default for other projects (1000) or removed if merged after #220

File: projects/portland-metro/.env [hunk 2]
```diff
@@ -1,3 +1,3 @@
 COMPOSE_PROJECT_NAME=pelias
-DATA_DIR=/tmp/pelias/portland-metro
-DOCKER_USER=1000
+DATA_DIR=./data
```
Comment (missinglink) [inline projects/portland-metro/.env:2]: This change seems unrelated, could you please revert it?

**Review by missinglink — CHANGES_REQUESTED** [review]
Looks good thanks, couple of small changes need to be made before merging.

File: projects/austria/.env (hunk 1 above)
Comment (jimmylevell) [reply projects/austria/.env]: Thank you very much for your quick review. As requested I have updated the docker_user to 1000.

File: projects/portland-metro/.env (hunk 2 above)
Comment (jimmylevell) [reply projects/portland-metro/.env]: Yes is unrelated. I have reverted the commit. Please review again.

File: projects/austria/.env (hunk 1 above)
Comment (missinglink) [reply projects/austria/.env]: Hey, sorry to be a pain but could you please remove that line 🙏 
We merged https://github.com/pelias/docker/pull/220 already and I've removed it from the other projects already.

Comment (jimmylevell) [reply projects/austria/.env]: oh sorry misunderstanding. Of course. As requested I have removed the line.

**Comment by missinglink** [comment]
Thanks!

File: projects/portland-metro/.env [hunk 3]
```diff
@@ -1,2 +1,3 @@
 COMPOSE_PROJECT_NAME=pelias
 DATA_DIR=/tmp/pelias/portland-metro
+DOCKER_USER=1000
```
Comment (missinglink) [inline projects/portland-metro/.env:3]: oops I missed this in review, fixed in https://github.com/pelias/docker/pull/224
